ndcube/asdf/resources/schemas/NDCube-0.1.0.yaml

```
@@ -0,0 +1,21 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/NDCube-0.1.0"
+
+title: |
+  Represents the ndcube NDCube object
+
+description:
+  Represents the ndcube NDCube object
+
+type: object
+properties:
+  data:
+    tag: "tag:stsci.edu:asdf/core/ndarray-1.0.0"
```

```suggestion
    tag: "tag:stsci.edu:asdf/core/ndarray-1.*"
```
`tag` keywords can use wildcards which can help to deal with tag versions. In this case [ASDF-standard 1.6.0](https://github.com/asdf-format/asdf-standard/blob/c367d14fae4b5155679e2c40b531140586e25daf/resources/manifests/asdf-format.org/core/core-1.6.0.yaml#L64) will introduce `ndarray-1.1.0`. By using a wilcard here this schema can work with both ASDF-standard <1.6.0 and =1.6.0.

---

Right, will do this.